src/SaveGameManager.h

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2008-2024 Pioneer Developers. See AUTHORS.txt for details
+// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+#ifndef _SAVEGAMEMANAGER_H
+#define _SAVEGAMEMANAGER_H
+
+#include "FileSystem.h"
+#include "JsonUtils.h"
```

Comment:
To keep compile times down we prefer to use forward declaration wherever possible, especially around the JSON library's very template-heavy code. Unless absolutely necessary, headers should include `JsonFwd.h` and their matching cpp files include `JsonUtils.h`.

Reply:
Fixed in 977cfc5aff4d61a352fbe153dbc42d8c4a619fa3